src/Backend/Modules/Faq/Resources/config/repositories.yml

```diff
@@ -0,0 +1,19 @@
+services:
+  faq.repository.category:
+    class: Backend\Modules\Faq\Domain\Category\CategoryRepository
+    factory: ["@doctrine.orm.entity_manager", getRepository]
+    arguments:
+      - Backend\Modules\Faq\Domain\Category\Category
+
+  faq.repository.question:
+    class: Backend\Modules\Faq\Domain\Question\QuestionRepository
+    factory: ["@doctrine.orm.entity_manager", getRepository]
+    arguments:
+      - Backend\Modules\Faq\Domain\Question\Question
+
+
```

Comment:
This empty line is obsolete and can be removed.

Reply:
Fixed 6bc39a695